src/katello/tracer.py

```diff
@@ -1,13 +1,29 @@
 from __future__ import absolute_import
```

Comment:
I see there are not any tests for tracer.py - now would be an excellent time to add one!

Reply:
working on it.

Reply:
I tried to add tests but to be honest, I currently don't know how to do the proper mocks.

Reply:
I'll take a look shortly :)